src/Hypergrid.js

```diff
@@ -2344,5 +2344,7 @@ Hypergrid.localization = {
 Hypergrid.prototype.mixIn(require('./lib/events'));
 Hypergrid.prototype.mixIn(require('./lib/selection'));
 
+Hypergrid.properties = globalProperties;
```

Comment:
This is just defaults.js, which is the "shared" global properties object. Alterations here affect all grids, including simple overrides and theming. `grid.properties` is extended (created) from this and these are the only base layers now (down from 5 or 6 layers). Additions to `grid.properties` override the global props.